src/mess/machine/fm_scsi.h

```diff
@@ -48,7 +48,7 @@ class fmscsi_device : public legacy_scsi_host_adapter
 	template<class _Object> static devcb_base &set_irq_handler(device_t &device, _Object object) { return downcast<fmscsi_device &>(device).m_irq_handler.set_callback(object); }
 	template<class _Object> static devcb_base &set_drq_handler(device_t &device, _Object object) { return downcast<fmscsi_device &>(device).m_drq_handler.set_callback(object); }
 
-	// any publically accessible interfaces needed for runtime
+	// any publicly accessible interfaces needed for runtime
```

Comment:
This is a legitimate although unusual spelling in US/Canadian English.  No need to change it.